instrumentation/github.com/syndtr/goleveldb/leveldb/splunkleveldb/README.md

```diff
@@ -0,0 +1,16 @@
+# Splunk instrumentation for `github.com/syndtr/goleveldb/leveldb`
+
+This package provides OpenTelemetry instrumentation for the
+[github.com/syndtr/goleveldb/leveldb](https://github.com/syndtr/goleveldb/leveldb)
+package.
+
+## Compatibility
+
+- Go: 1.15+
+- github.com/syndtr/goleveldb/leveldb: 1.0.0+
+
+## Getting Started
+
+This package is designed to be used as a drop-in replacement for the use of the
+`github.com/syndtr/goleveldb/leveldb` package. Examples of how to do so can be
+found [here](./example_test.go).
```

Comment:
Suggested change:  "Examples of how to do so can be
found [here](./example_test.go)." to "See example_test.go for more information."